Style.css

```diff
@@ -459,6 +459,10 @@ textarea::placeholder {
   margin-top: 515px;
 }
 
+.popup-d {
+  display: none;
+}
+
```

Comment:
- [ ] Good job implementing different layouts for mobile and desktop versions:100: :+1:, but as a project requires the popup should be responsive, it means should fit mobile and desktop sizes properly.
- [ ] While you did a fantastic job styling your popup, I think it would be better if you stick as much as possible to the Figma template, it means making sure that your popup is placed in the middle of your page, also check if the background is the same and finally fix the elements that are been overlapping :+1: 
- [ ] As required for the Figma, the mobile version should have the buttons "See live" and "See source" at the end of the popup and the 'x' button at the right :+1: 

**Your design**

![Screenshot from 2023-02-08 05-49-37](https://user-images.githubusercontent.com/105117832/217509777-27f298d9-c52b-4404-ba4b-401a5a8e8724.png)

![Screenshot from 2023-02-08 05-50-45](https://user-images.githubusercontent.com/105117832/217509810-f4bce19d-f168-4103-b59c-19aef2e2e7d2.png)

**Figma**

![Screenshot from 2023-02-08 05-54-36](https://user-images.githubusercontent.com/105117832/217510142-874818c4-317b-4b6a-ac22-23b538f1a060.png)

![Screenshot from 2023-02-08 05-54-52](https://user-images.githubusercontent.com/105117832/217510197-1a653978-95d9-4ab7-8aad-da87d4791b83.png)